docs/frontend/architecture.md

```
@@ -4,248 +4,255 @@ This document explains the architecture and structure of the PhiloBiblon UI fron
-- SEO is not a primary concern (internal tool)
+```typescript
+// nuxt.config.ts
+export default defineNuxtConfig({
+  ssr: false,
+  ...
+})
+```
 
 ## Directory Structure
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)